Modules/rat_rescue.py

```diff
@@ -577,12 +606,12 @@ def open(self, value: bool) -> None:
             raise TypeError(f"expected type bool, got {type(value)}")
 
     @property
-    def epic(self) -> bool:
+    def epic(self) -> Epic:
```

Comment:
The `epic` property should be a list (or set or tuple) of `Epic` objects, I believe a rescue can have multiple epics.

Reply:
ah right, i overlooked that >.>